.vscode/launch.json

```diff
@@ -20,7 +20,8 @@
       "type": "node",
       "protocol": "inspector",
       "env": {
-        "DEBUG": "1"
+        "DEBUG": "1",
+        "NODE_OPTIONS": "--dns-result-order=ipv4first",
```

Comment:
A note for posterity: This change is for an error we encountered when using the VSCode Debugger with Functional Tests.

The error was: `RequestError: connect ECONNREFUSED ::1:9001`

This is related to a change in default behavior in Node 17 when we upgraded from v16 to v18 recently per @chenba .